### PR TITLE
Itemsets, Association: autorun when checkbox

### PR DESCRIPTION
**Pull request by jerneju**
##### Issue
See https://github.com/biolab/orange3-associate/pull/12#issuecomment-333760426.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: orangecontrib/associate/widgets/owassociate.py [hunk 1]
```diff
@@ -96,7 +96,8 @@ def __init__(self):
         self.cb_classify = gui.checkBox(
             box, self, 'classify', label='Induce classification (itemset → class) rules')
         self.button = gui.auto_commit(
-                box, self, 'autoFind', 'Find rules', callback=self.find_rules)
+                box, self, 'autoFind', 'Find rules', commit=self.find_rules,
+                callback=self.find_rules_checked)
```
Comment (kernc) [inline orangecontrib/associate/widgets/owassociate.py]: This is work?

Comment (jerneju) [reply orangecontrib/associate/widgets/owassociate.py]: Thanks! Now it works.

**Comment by kernc** [comment]
@jerneju Please review the change I made in your name.

File: orangecontrib/associate/widgets/owassociate.py [hunk 2]
```diff
@@ -130,7 +130,8 @@ def selectionChanged(self, selected, deselected):
         self.cb_classify = gui.checkBox(
             box, self, 'classify', label='Induce classification (itemset → class) rules')
         self.button = gui.auto_commit(
-                box, self, 'autoFind', 'Find rules', callback=self.find_rules)
+                box, self, 'autoFind', 'Find rules', commit=self.find_rules,
+                callback=lambda: self.autoFind and self.find_rules())
```
Comment (jerneju) [inline orangecontrib/associate/widgets/owassociate.py]: Great!

**Pull request merged**